### ISSUE TITLE
ASE calculate() method should have properties and system_changes arguments

### ISSUE THREAD

**Issue by mhellstr** [CederGroupHub/chgnet#34]
The CHGNetCalculator in model/dynamics.py is defined with

```py
    def calculate(
        self,
        atoms: Atoms | None = None,
        desired_properties: list | None = None,
        changed_properties: list | None = None,
    ) -> None:
```

To be ASE compatible ([`ase/calculators/calculator.py`](https://wiki.fysik.dtu.dk/ase/development/calculators.html)) it would be great if it could support the `properties` and `system_changes` args.

```py
def calculate(self, atoms=None, properties=['energy'],
                  system_changes=all_changes):
```

**Comment by BowenD-UCB** [comment]
Hi, 

The `desired_properties` and `changed_properties` are fed to the ASE base calculator. 
Please refer here:
https://github.com/CederGroupHub/chgnet/blob/95976d69843b39ac28630616159c751fb931d3a3/chgnet/model/dynamics.py#L87-L108

But these two keys are redundant for CHGNet calculation because in the current settup, CHGNet will always calculate the energy, forces, stress and magmoms in the calculator.
Can you provide more information on how you want to use the `properties` and `system_changes` that current CHGNet calculator can not do?

**Comment by mhellstr** [comment]
Hi,

the problem is that some code (for example my own...) may assume that the arguments are called ``properties`` and ``system_changes``. So it would be nice if it would be possible to just rename them to be consistent with the rest of ASE.

By the way, for most of the test cases I've played with CHGNet seems to work great!

**Comment by BowenD-UCB** [comment]
Thanks for this comment, it should be a valueble change.
This is fixed with commit 628c62fe087907fc3e3d2a67d3642861706833a9
